### ISSUE TITLE
Rule `sysmon_password_dumper_lsass.yml` does not contain value for field

### ISSUE THREAD

**Issue by barvhaim** [Neo23x0/sigma#945]
https://github.com/Neo23x0/sigma/blob/32ecb816307e3639cf815851fac8341a60631f45/rules/windows/sysmon/sysmon_password_dumper_lsass.yml

Field `StartModule` does not have a value, is it on purpose or by mistake?

**Comment by Neo23x0** [comment]
Yes, it should be empty, but the way to describe it could be wrong. I think we had a `null` value for fields that don't exist fields and `` for empty strings.

**Comment by frack113** [comment]
Sorry this post is closed automatically because it is not more active
